samples/snippets/create_kmeans_model_test.py

```diff
@@ -41,21 +44,30 @@ def test_kmeans_sample(project_id: str, random_model_id_eu: str) -> None:
         }
     )
 
-    s = bpd.read_gbq(
-        # Use ST_GEOPOINT and ST_DISTANCE to analyze geographical
-        # data. These functions determine spatial relationships between
-        # geographical features.
-        """
-        SELECT
-        id,
-        ST_DISTANCE(
-            ST_GEOGPOINT(s.longitude, s.latitude),
-            ST_GEOGPOINT(-0.1, 51.5)
-        ) / 1000 AS distance_from_city_center
-        FROM
-        `bigquery-public-data.london_bicycles.cycle_stations` s
-        """
+    # Use GeoSeries.from_xy and BigQuery.st_distance to analyze geographical
+    # data. These functions determine spatial relationships between
+    # geographical features.
+    # See, https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.geopandas.GeoSeries#bigframes_geopandas_GeoSeries_from_xy
+    # and https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.bigquery?_gl=1*zvtewl*_ga*MTQ0Mjc3NTA5NS4xNzQyOTI5NDMz*_ga_4LYFWVHBEB*MTc0NTg1OTY2Mi4zLjEuMTc0NTg2MDQ2Mi4wLjAuMA..#bigframes_bigquery_st_distance.
```

Comment:
Let's clean this link up a bit.

```suggestion
    # and https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.bigquery#bigframes_bigquery_st_distance.
```

Reply:
SG! I'm noticing the current public API docs are not reflecting the changes you made in https://github.com/googleapis/python-bigquery-dataframes/pull/1637. I'll try re-staging the docs.

Reply:
It seems we are waiting for issue 414395527 to be fixed. Removing the links because the `st_distance` documentation is has not been staged yet.